src/Microsoft.DotNet.Configurer/xlf/LocalizableStrings.ja.xlf

```diff
@@ -16,7 +16,7 @@ You can read more about .NET Core tools telemetry @ https://aka.ms/dotnet-cli-te
 
 Configuring...
 -------------------
-A command is running to initially populate your local package cache, to improve restore speed and enable offline access. This command will take up to a minute to complete and will only happen once.</source>
+A command is running to populate your local package cache. This will improve restore speed and enable offline access. It will take up to a minute to complete and will only happen once.</source>
         <target state="translated">.NET Core へようこそ!
```

Comment:
I realize now that my tool hadn't been integrated yet. Now that it is, revert xlf, merge and then build with UpdateXlfOnBuild=true. The xlf will be automatically updated correctly based on the resx.